lib/core/src/Cardano/Wallet/Api.hs

```diff
@@ -313,6 +314,13 @@ type SelectCoins n = "wallets"
     :> ReqBody '[JSON] (ApiSelectCoinsDataT n)
     :> Post '[JSON] (ApiCoinSelectionT n)
 
+type SelectCoinsAction n = "wallets"
```

Comment:
I forgot if we want to keep this or extend `ApiSelectCoinsData` instead?

```hs
newtype ApiSelectCoinsData (n :: NetworkDiscriminant) = ApiSelectCoinsData
    { payments :: NonEmpty (AddressAmount (ApiT Address, Proxy n))
    , action :: Maybe DelegationAction
    } deriving (Eq, Generic, Show)
```

something like that?